src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts

```diff
@@ -68,11 +69,16 @@ export abstract class SimpleFindWidget extends Widget {
 		// Find History with update delayer
 		this._updateHistoryDelayer = new Delayer<void>(500);
 
-		this.oninput(this._findInput.domNode, (e) => {
-			this.foundMatch = this._onInputChanged();
-			this.updateButtons(this.foundMatch);
-			this._delayedUpdateHistory();
-		});
+		this._register(this._findInput.onInput((e) => {
+			if (checkImeCompletionState && !this._findInput.isImeSessionInProgress) {
```

Comment:
Note to self before merge: I'm not sure whether this condition is correct.
I think it should be that if we're checking for the completion state, then we check if the session is not in progress.

Reply:
Fixed in latest commit. Also removed empty lines in this block.